src/main/java/com/cdancy/jenkins/rest/fallbacks/JenkinsFallbacks.java

```diff
@@ -153,7 +153,7 @@ public static SystemInfo createSystemInfoFromErrors(final List<Error> errors) {
         final List<Error> errors = Lists.newArrayList();
 
         String context = null;
-        String message = null;
+        String message = output.getMessage();
```

Comment:
We shouldn't need to do this now that we have the @Nullable annotation.

Reply:
This will provide some support for exceptions whose messages do not parse with "->" below.

Reply:
Fair enough.